x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -595,4 +595,35 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.alerts.cloud_lookup',
+    first_supported_version: '7.12',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.alerts.cloud_lookup',
+      {
+        defaultMessage:
+          "A value of 'false' disables cloud lookup for Windows alerts. Default: true.",
```

Comment:
since you quoted 'false`, maybe also quote `true`?
(same comment in other areas below

Reply:
at least at the moment, it's consistent with the other existing entries.  I'll leave it for now.